acme/order.go

```diff
@@ -45,10 +45,12 @@ func (o *Order) GetID() string {
 
 // OrderOptions options with which to create a new Order.
 type OrderOptions struct {
-	AccountID   string       `json:"accID"`
-	Identifiers []Identifier `json:"identifiers"`
-	NotBefore   time.Time    `json:"notBefore"`
-	NotAfter    time.Time    `json:"notAfter"`
+	AccountID       string        `json:"accID"`
+	Identifiers     []Identifier  `json:"identifiers"`
+	NotBefore       TimeDuration  `json:"notBefore"`
+	NotAfter        TimeDuration  `json:"notAfter"`
```

Comment:
We should use `time.Time` to avoid the extension of the spec by being able to use durations too.